### ISSUE TITLE
[Perception] Scene 3 - Screen Reader Functionality

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-people-use-web-videos#100]
I am a little worried that in Scene 3 we may receive some criticism for the sentence with the following text "... These software tools capture information on the screen and convert it to..." For those out there who are going to listen to the script in a literal way, the screen reader doesn't really do that. It reads the code and translates what it needs into speech. It doesn't read the screen. 

I know we all know this, but I am worried about this wording. Maybe we don't even need to get this detailed to avoid the literal functionality. 

Just wanted to bring this up in case others may agree.

**Comment by nitedog** [comment]
Changed:

> These software tools capture information on the screen and convert it to audio to hear, or convert it to braille letters on a refreshable braille display.

To:

> These software tools present information on the screen in audio to hear, or in braille on a refreshable braille display.
